app/src/main/java/de/danoeh/antennapod/adapter/DefaultActionButtonCallback.java

```diff
@@ -81,9 +81,11 @@ public void onActionButtonPressed(final FeedItem item, final LongList queueIds)
                 }
             } else { // media is downloaded
                 if (item.hasMedia() && item.getMedia().isCurrentlyPlaying()) {
+                    PlaybackService.startIfNotRunning(context, media, true, false);
```

Comment:
can we extract the arguments into an object with a builder?
IntelliJ/AS will tell you what these booleans mean, but this should be readable without the help of the IDE